### DISCUSSION TITLE
How to use the station as well as the exit/entry rail for them?

### DISCUSSION THREAD

**Discussion by Xiarno** [Opinionated-Blueprints/10-Books-Full-of-Rails#6] [Q&A] [answered]
I'm trying to use the book to make some station. My understanding is that the book was designed so you only have to use the book and not place thing manually, so it stays consistent. So how do I use the station and entry/exit rail to make... well, just that?

Vanilla, latest Factorio version.

**Comment by rpdelaney** [comment, marked as answer]
My assumption, when I work with this book, is that an exit/entry will be connected shortly thereafter to a stacker or station. The exact positioning of these features should be determined by the engineer on the ground according to their specific requirements in that locality. Therefore, exits/entrances aren't aligned to the main grid.
